src/test/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractValidationServiceImplTest.java

```diff
@@ -116,5 +141,12 @@ private void setBadDebitCredit() {
     private void setBadRecordCount() {
         file.setRecordCount(new Integer(5));
     }
+    
+    private class TestableConcurStandardAccountingExtractValidationServiceImpl extends ConcurStandardAccountingExtractValidationServiceImpl {
+        @Override
+        protected String findEmploueeGroupId() {
+            return "CORNELL";
```

Comment:
It might make sense to create a testing-only constant for this value, given that at least one other test class relies on it (and the unit tests for my story may need it as well).